### PR TITLE
fix: add <NikhilRaikwar>

### LINKED ISSUE #1729: [PROFILE ADDITION] <Nikhil Raikwar>
### Profile screenshot

Screenshot 2023-02-15 223556

### PR DESCRIPTION
**Pull request by NikhilRaikwar**
resolves #1729 

## Checklist for profile addition

- [ ] Name the PR `fix: add <your-username>`.
- [ ] You've added file named `your-username.json` in the [profiles](https://github.com/EddieHubCommunity/awesome-github-profiles/tree/main/profiles) directory.
- [ ] You've linked the PR to the correct issue.

---

## Other additions/changes

- [ ] Changes made in this PR are mentioned below
- [ ] Search previous suggestions before making a new one, as yours may be a duplicate.
- [ ] I acknowledge that all my contributions will be made under the project's license.
- [ ] I've read the [contributing guidelines](https://github.com/EddieHubCommunity/awesome-github-profiles/blob/main/CONTRIBUTING.md)

### CODE REVIEW

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Comment by NikhilRaikwar** [comment]
hey can i know when was my pr are going to be merged?

**Comment by Panquesito7** [comment]
> hey can i know when was my pr are going to be merged?

It will be merged once two or more maintainers approve the PR. 🙂

**Review by schmelto — APPROVED** [review]

**Pull request merged**
